tests/contenttypes_tests/tests.py

```diff
@@ -439,6 +462,31 @@ def test_interactive_false(self):
         self.assertIn("Stale content types remain.", stdout.getvalue())
         self.assertEqual(ContentType.objects.count(), self.before_count + 1)
 
+    def test_force_remove_interactive_false(self):
+        """
+        non-interactive and non-force_remove modes of update_contenttypes()
+        shouldn't delete stale content types.
+        """
+        with captured_stdout() as stdout:
+            management.update_contenttypes(self.app_config, interactive=False, force_remove=False)
+        self.assertIn("Stale content types remain.", stdout.getvalue())
+        self.assertEqual(ContentType.objects.count(), self.before_count + 1)
+
+
+class RemoveContentTypesTests(TestCase):
+    def setUp(self):
+        self.before_count = ContentType.objects.count()
+        ContentType.objects.create(app_label='contenttypes_tests', model='Fake')
+        self.app_config = apps.get_app_config('contenttypes_tests')
+
+    def test_remove_contenttypes(self):
+        """
+        remove_contenttypes should call update_contenttypes.
+        """
+        with mock.patch('django.contrib.contenttypes.management.update_contenttypes') as uct:
+            management.remove_contenttypes(self.app_config)
+            uct.assert_called_times, 4
```

Comment:
`self.assertEqual(uct.call_count, 4)`
But IMO it is better to use patch as decorator instead of context manager.